lib/ex_doc/formatter/html/templates/redirect_template.eex

```diff
@@ -0,0 +1,11 @@
+<%= head_template(%{id: :redirect, content: config, redirect_to: redirect_to, }) %>
```

Comment:
I think that, instead of passing all those options, we should just generate the redirect template directly here, including the html head. It will make this and all other templates cleaner. What do you think?